python/GafferUI/SpreadsheetUI/_ProxyModels.py

```diff
@@ -40,6 +40,15 @@
 
 class __PlugTableProxyModel( QtCore.QAbstractProxyModel ) :
 
+	def __init__( self, startRow = 0, startColumn = 0, rowCount = None, columnCount = 0, parent = None ) :
+
+		QtCore.QAbstractProxyModel.__init__( self, parent = None )
```

Comment:
`parent = parent`

Reply:
Thanking you! [Fixed](https://github.com/GafferHQ/gaffer/pull/3995/commits/ff2b0127b96ce197cc5dd39b95aa8be0fadda4e3).